.travis.yml

```diff
@@ -1,3 +1,8 @@
+sudo: required
```

Comment:
Can you try using `pip install --user` and see if that works around the need for sudo?